extension/chrome/settings/modules/backup.ts

```diff
@@ -15,7 +15,10 @@ import { BackupManualActionModule as BackupManualModule } from './backup-manual-
 import { BackupAutomaticModule } from './backup-automatic-module.js';
 import { Lang } from '../../../js/common/lang.js';
 import { AcctStore, EmailProvider } from '../../../js/common/platform/store/acct-store.js';
-
+export interface PrvKeyIdentity {
```

Comment:
It's a good point to have a `KeyIdentity` type (likely in `extension/js/common/core/crypto/key.ts`).
There are already a couple of places in the codebase that utilize a structure like `{ id: string, type: 'openpgp' | 'x509' }` -- that's the perfect candidate to be `KeyIdentity`. There are plenty of places in the codebase that use `fingerprints[0]`, it's hard to follow whether they assume `fingerprints[0]` to be unique within the retrieved records or not. I see some potential problems, should we happen have (although a very rare case) both `x509` and `openpgp` keys with the same fingerprint -- for example, with different passphrases. It might be worthwhile to introduce functions that return explicitly typed identity, and gradually phase out usage of `fingerprints[0]` where appropriate.
Are you ok to go ahead with such refactoring, @tomholub

Reply:
>for example, with different passphrases. It might be worthwhile to introduce functions that return explicitly typed identity, and gradually phase out usage of fingerprints[0] where appropriate.

Thank you, sir! I just move/rename and use the KeyIdentity type. I wasn't able to follow the preceding comments yet in this thread. But I managed to add the recent request change.